src/sentry/options/defaults.py

```diff
@@ -10,15 +10,38 @@
 from sentry.options import (
     FLAG_IMMUTABLE, FLAG_NOSTORE, FLAG_PRIORITIZE_DISK, FLAG_REQUIRED, register
 )
-from sentry.utils.types import Dict, String
+from sentry.utils.types import Dict
 
+
+def default_from_email():
```

Comment:
I'm not entirely sure. I know anytime I've poked at this, `root@localhost` 100% of the time went into spam, but if I changed it to my local hostname, it would get delivered.

I'll investigate and confirm this behavior. Note though, the root@localhost issue only seemed to exist with `[Sentry]` emails. So I'm guessing Gmail sees this combo and assumes spam since it's probably pretty widespread and people aren't setting this default. Which, itself, is another problem. Since we should encourage setting this in the install wizard.